File: eclipse-plugins/edu.wpi.first.wpilib.plugins.java/resources/templates/examples/PacGoat/src/org/usfirst/frc/team190/pacgoat/commands/LowGoal.java
package $package.commands;

import $package.subsystems.Collector;
import $package.subsystems.Pivot;

import edu.wpi.first.wpilibj.command.CommandGroup;

/**
 * Spit the ball out into the low goal assuming that the robot is in front of it.
 */
public class LowGoal extends CommandGroup {
    public  LowGoal() {
    	addSequential(new SetPivotSetpoint(Pivot.LOW_GOAL));
    	addSequential(new SetCollectionSpeed(Collector.REVERSE));
    	addSequential(new ExtendShooter());
    }
}

File: eclipse-plugins/edu.wpi.first.wpilib.plugins.cpp/resources/templates/examples/PacGoat/src/Commands/LowGoal.cpp
#include "LowGoal.h"

#include "Robot.h"
#include "Commands/SetPivotSetpoint.h"
#include "Commands/SetCollectionSpeed.h"
#include "Commands/ExtendShooter.h"

LowGoal::LowGoal() {
	AddSequential(new SetPivotSetpoint(Pivot::LOW_GOAL));
	AddSequential(new SetCollectionSpeed(Collector::REVERSE));
	AddSequential(new ExtendShooter());
}
